Quick refactor and Prototype switching pdf

gezgblack opened the pull request:
This is a small PR, in this PR I have just centered the contents of the index page and I have changed the Bootstrap path in the `pdfExperiment.html` as it was wrong. 

Also I have added a JavaScript function and a button to change between the question and answer papers, we will discuss the change of this function to angular `ng-show` and putting it into the appropriate js file and tests for it tomorrow ;) we can merge after the fix too.

![096742da-c99c-4fd2-8329-4d4881ba5e17](https://cloud.githubusercontent.com/assets/7914065/17872129/8edc35ee-68b6-11e6-82f5-471240fd8e5e.gif)

@Sukrat

Sukrat commented on app/index.html:
```diff
@@ -31,7 +31,7 @@
 
 <body ng-app="xamApp">
-    <div class="container">
+    <div class="text-center">
```
I think we should do this text-center inside the container. As this deviates from the bootstrap way and might not work on phones properly then.

Sukrat commented on app/pdfExperiment.html:
```diff
@@ -3,19 +3,40 @@
+
+    console.log(container.innerHTML);
+}
+</script>
```
This is exactly the logic we want! But the thing is we are switching the url. It will reload the pdf again and again and will make the app slow. What i would suggest to load both at page load and then switch the display to none and vice versa! But yeah thats what we want! Also it will be directive!

Sukrat commented:
Overall good job dude! The main thing we were stuck is solved. Thanks to you. I ll work on the crawler till then you can start turning it into angular! :+1:

gezgblack commented:
@Sukrat right dude :) I have made it in a directive, I have changed the pdf so now it does not reload the PDF every single time. I just created an overlay, I overlaid the answer papers over the question papers and I hid them. And I added a button which change the `visibility` of the overlaid paper from `hidden `to `visible`. Also the new CSS I added is necessary, there's no way around it :/. The only thing I have questions about is the grunt, we will chat about it. but yeah have a look :)

Sukrat commented on app/index.html:
```diff
@@ -31,7 +31,7 @@
 
 <body ng-app="xamApp">
-    <div class="container">
+    <div class="container text-center">
```
this one :+1:

Sukrat commented on app/pdfController.js:
```diff
@@ -0,0 +1,18 @@
+angular.module('MyModule', [])
+
+.directive('visible', function() {
```
the directive should do everything!

Sukrat commented on app/pdfController.js:
```diff
@@ -0,0 +1,18 @@
+  };
+})
+
+.controller('MyController', function($scope) {
```
No need to create separate controller! You can add controller to directive itself!

Sukrat commented:
@gezgblack I am merging this commit! But we will have to work on this again! Otherwise nice work!
